department-of-veterans-affairs/caseflow
# adding a check to see if the docket date is more than 30 days away in HP

Connects #4823 

### Description
a quick UI check for dates

### Acceptance Criteria 
- [ ] If a docket date is more than 30 days away, there should be no link on your hearing days page.

### Testing Plan
1. Run `rake local:vacols:seed` in the console
2. Check the 'Your hearing Days' page and notice couple of clickable links
